app/jobs/iiif_print/jobs/child_works_from_pdf_job.rb

```diff
@@ -58,17 +60,18 @@ def perform(candidate_for_parency, pdf_paths, user, admin_set_id, *)
       # rubocop:disable Metrics/ParameterLists
       # rubocop:disable Metrics/MethodLength
       def split_pdf(original_pdf_path, user, child_model, pdf_file_set)
-        image_files = @parent_work.iiif_print_config.pdf_splitter_service.call(original_pdf_path, file_set: pdf_file_set)
+        user = User.find_by_user_key(user) unless user.is_a?(User)
+        image_files = @parent_work.iiif_print_config.pdf_splitter_service.call(original_pdf_path)
 
         # give as much info as possible if we don't have image files to work with.
         if image_files.blank?
-          raise "#{@parent_work.class} (ID=#{@parent_work.id} " /
-                "to_param:#{@parent_work.to_param}) " /
-                "original_pdf_path #{original_pdf_path.inspect} " /
+          raise "#{@parent_work.class} (ID=#{@parent_work.id} " \
+                "to_param:#{@parent_work.to_param}) " \
+                "original_pdf_path #{original_pdf_path.inspect} " \
                 "pdf_file_set #{pdf_file_set.inspect}"
         end
 
-        @split_from_pdf_id = pdf_file_set&.id
+        @split_from_pdf_id = pdf_file_set&.id.to_s
```

Comment:
This shifts `@split_from_pdf_id` from a possible `nil` to an `empty` string.  I assume that's intended.